.rubocop.yml

```diff
@@ -58,17 +74,16 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
-Metrics/AbcSize:
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/SpaceAroundEqualsInParameterDefault:
   Enabled: false
 
-Metrics/LineLength:
+# As of July 2016, there is enough disagreement in our codebase, on our team,
+# and apathy to the rule to suggest a negative cost-benefit of enabling the
+# rule.
+Style/StringLiterals:
```

Comment:
For now, though, I don't think this rule belongs in the todo. E.g., if this were the last rule remaining, I'm not sure we would want to fix it (my bar for moving it to this file).